src/features/game/types/craftables.ts

```diff
@@ -1201,7 +1201,7 @@ export const COLLECTIBLES_DIMENSIONS: Record<CollectibleName, Dimensions> = {
   "Tin Turtle": { height: 1, width: 1 },
   "Basic Scarecrow": { width: 1, height: 1 },
   Bale: { width: 2, height: 2 },
-  "Sir Goldensnout": { width: 2, height: 2 },
+  "Sir Goldensnout": { width: 1, height: 2 },
```

Comment:
If we change the width of sir goldensnout, would it affect its AOE effect?

Reply:
Ahhh yeah potentially. I will remove these changes for now.